pool/hub.go

```diff
@@ -449,8 +450,19 @@ func (h *Hub) getBlockConfirmations(ctx context.Context, hash *chainhash.Hash) (
 	info, err := h.nodeConn.GetBlockVerbose(ctx, hash, false)
 	if err != nil {
 		desc := fmt.Sprintf("unable to fetch block confirmations: %v", err)
-		return 0, errs.PoolError(errs.BlockConf, desc)
+
+		var rErr dcrjson.RPCError
```

Comment:
```suggestion
		var rErr *dcrjson.RPCError
```

Should be a pointer for `errors.As`